### PR TITLE
Remove duplicated 'Timestamp cache' entries

### LINKED ISSUE #4124: Duplicate documentation for Timestamp cache
In documentation for `Transaction Layer`, the section for Timestamp cache is repeated twice, with no difference in documentation

Full path to docs : `Concepts -> Architecture -> Transaction Layer`. Attached below are the highlighted sections in a screenshot

screen shot 2018-12-02 at 5 53 36 pm

### PR DESCRIPTION
**Pull request by rmloveland**
Fixes #4124.

### CODE REVIEW

File: v2.0/architecture/transaction-layer.md [hunk 1]
```diff
@@ -80,9 +80,9 @@ For more detail about the risks that large clock offsets can cause, see [What ha
 
 ### Timestamp Cache
 
-To provide serializability, whenever an operation reads a value, we store the operation's timestamp in a Timestamp Cache, which shows the high-water mark for values being read.
+To provide serializability, whenever an operation reads a value, we store the operation's timestamp in a timestamp cache, which shows the high-water mark for values being read.
 
-Whenever a write occurs, its timestamp is checked against the Timestamp Cache. If the timestamp is less than the Timestamp Cache's latest value, we attempt to move the timestamp for its transaction forward to a later time. In the case of serializable transactions, this causes them to restart in the second phase of the transaction.
+Whenever a write occurs, its timestamp is checked against the timestamp cache. If the timestamp is less than the timestamp cache's latest value, we attempt to push the timestamp for its transaction forward to a later time. In the case of serializable transactions, this might cause them to restart in the second phase of the transaction (see [read refreshing](#read-refreshing)).
```
Comment (sploiselle) [inline v2.0/architecture/transaction-layer.md]: Did you get feedback about the "causes" vs. "might cause" technical detail?

File: v2.1/architecture/transaction-layer.md [hunk 2]
```diff
@@ -84,7 +84,7 @@ For more detail about the risks that large clock offsets can cause, see [What ha
 
 To provide serializability, whenever an operation reads a value, we store the operation's timestamp in a timestamp cache, which shows the high-water mark for values being read.
 
-Whenever a write occurs, its timestamp is checked against the timestamp cache. If the timestamp is less than the timestamp cache's latest value, we attempt to move the timestamp for its transaction forward to a later time. In the case of serializable transactions, this causes them to restart in the second phase of the transaction.
+Whenever a write occurs, its timestamp is checked against the timestamp cache. If the timestamp is less than the timestamp cache's latest value, we attempt to push the timestamp for its transaction forward to a later time. In the case of serializable transactions, this might cause them to restart in the second phase of the transaction (see [read refreshing](#read-refreshing)).
```
Comment (sploiselle) [inline v2.1/architecture/transaction-layer.md]: Doh. This is a place that we should update to clarify that all transactions are serializable. Do you mind if I piggy-back the edits on this PR, or you want to merge this and I'll open a new one?

File: v2.0/architecture/transaction-layer.md (hunk 1 above)
Comment (rmloveland) [reply v2.0/architecture/transaction-layer.md]: Nope! I just went with the one that looked slightly "righter".

Looking at `git blame` it looks I got lucky and @andreimatei authored the replacement text in 73fa655

File: v2.1/architecture/transaction-layer.md (hunk 2 above)
Comment (rmloveland) [reply v2.1/architecture/transaction-layer.md]: Feel free to piggy-back / take this one over.  Thanks!

**Pull request merged**